dss-pades-pdfbox/src/main/java/eu/europa/esig/dss/pdf/pdfbox/visible/nativedrawer/NativePdfBoxVisibleSignatureDrawer.java

```diff
@@ -105,6 +116,17 @@ private PDFont initFont() throws IOException {
 		}
 	}
 
+	/**
+	 * Method to initialize the sRGB ICC profile for PdfBox {@link PDOutputIntent}
+	 */
+	private PDOutputIntent initOutputIntent(PDDocument document) throws IOException {
+		ICC_Profile iccProfile = ICC_Profile.getInstance(ColorSpace.CS_sRGB);
+		PDOutputIntent outputIntent = new PDOutputIntent(document, new ByteArrayInputStream(iccProfile.getData()));
+		outputIntent.setOutputCondition(OUTPUT_INTENT_SRGB_PROFILE);
```

Comment:
How does this work when a document already has a CalGray or DeviceCMYK colorspace? If I'm correct PDF/A does not allow multiple colorspaces in the same document.
Does this invalidate existing signatures?

Reply:
I've tested this myself now and it works. I get a signed document which is PDF/A compliant and has 2 colorspaces. It also doesn't invalidate any existing signatures/timestamps.

I think we should merge this ASAP.

Reply:
Yeah, I think the output intent itself shouldn't have impact on existing colorspaces.
Also it doesn't invalidate existing signatures, because the output intent is part of the PDF container itself and outside the signed byte range. It's been a while since I opened this PR, so I guess I'll have to update it first.

Reply:
Yes, please do update it so it can be merged ASAP.